flake.nix

```diff
@@ -75,6 +108,23 @@
           inherit (self) outputs;
         in
         {
+          nixosConfigurations = {
+            homestakeros = nixpkgs.lib.nixosSystem {
+              system = "x86_64-linux";
+              specialArgs = { inherit inputs outputs; };
+              modules = [
+                nixobolus.nixosModules.kexecTree
+                nixobolus.nixosModules.homestakeros
+                ./nixosConfigurations/homestakeros
+                {
+                  system.stateVersion = "23.05";
+                  # Bootloader for x86_64-linux / aarch64-linux
+                  boot.loader.systemd-boot.enable = true;
+                  boot.loader.efi.canTouchEfiVariables = true;
+                }
+              ];
+            };
+          };
```

Comment:
This block also should be modified so that the hosts are defined by the basis of ls command to the nixosConfigurations folder, not by hardcoded values.

Reply:
Further, it is worth noting that this should eventually support both .ISO and kexec tree generation. For now, it's fine to continue with kexec only support, but this requirement suggests that we should eventually have a homestakeros option for the boot media configuration  as well.

Reply:
> This block also should be modified so that the hosts are defined by the basis of ls command to the nixosConfigurations folder, not by hardcoded values.

Yeah, I just remembered that I created a function for this exact purpose a while back and it will fit perfectly here.